tests/test-sync-request.js

```diff
@@ -7,112 +7,113 @@ chai.use(dirtyChai)
 const expect = chai.expect
 const XMLHttpRequest = require('../lib/XMLHttpRequest').XMLHttpRequest
 
-describe('XMLHttpRequest sync request', () => {
```

Comment:
The file is named `test-sync-request.js`. If we want to test both sync and async requests we should rename it. Alternatively we could create a dedicated file for async requests (and share some code between the two tests suite).

Reply:
yes. I'm also trying to enable https testing.